check.py

```diff
@@ -14,30 +14,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os, shutil, sys, subprocess, difflib, json, time, urllib2
+import os, shutil, sys, subprocess, difflib, json, time, urllib2, optparse
 
 import scripts.storage
 from scripts.support import run_command, split_wast
 
-interpreter = None
-requested = []
-torture = True
-only_prepare = False
-
-for arg in sys.argv[1:]:
-  if arg.startswith('--interpreter='):
-    interpreter = arg.split('=')[1]
-    print '[ using wasm interpreter at "%s" ]' % interpreter
-    assert os.path.exists(interpreter), 'interpreter not found'
-  elif arg == '--torture':
-    torture = True
-  elif arg == '--no-torture':
-    torture = False
-  elif arg == '--only-prepare':
-    only_prepare = True
-  else:
-    requested.append(arg)
-
+usage_str = "usage: 'python check.py [options]'\n\n Runs the Binaryen test suite."
+parser = optparse.OptionParser(usage=usage_str)
+parser.add_option('--torture', dest='torture', action='store_true', default=True, help='Chooses whether to run the torture testcases. Default: true.')
```

Comment:
It seems unnecessary to have these options which are just on by default. It seems like the only use case would be just to allow them to appear after the negation versions to turn the option back on again?